public/kcl-samples/manifest.json

```diff
@@ -3,29 +3,29 @@
     "file": "main.kcl",
     "pathFromProjectDirectoryToFirstFile": "80-20-rail/main.kcl",
     "multipleFiles": false,
-    "title": "80/20 Rail",
-    "description": "An 80/20 extruded aluminum linear rail. T-slot profile adjustable by profile height, rail length, and origin position"
+    "title": "@settings(defaultLengthUnit = in)",
```

Comment:
We need to fix this before merging. This shows up in ZMA in the command bar when you open a sample.

Reply:
yeah i mentioned something to @nrc in slack becausse i tried to fix but yeah this is basically showing why the ML team doesnt format anymore :(